x-pack/plugins/security/server/authentication/providers/kerberos.ts

```diff
@@ -264,12 +267,12 @@ export class KerberosAuthenticationProvider extends BaseAuthenticationProvider {
       return AuthenticationResult.failed(err);
     }
 
-    // If refresh token is no longer valid, then we should clear session and renegotiate using SPNEGO.
+    // If refresh token is no longer valid, let's check if the request is allowed to start a new session, and if so try
```

Comment:
Is this comment still valid? It seems like we don't have to check if the request is allowed to start a new session anymore, since authentication via refresh token is extending an existing session.

Reply:
You're absolutely right! This looks like a leftover from one of my initial WIPs, will fix, thanks!